charts/jaeger-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.18.1
+version: 2.18.2
```

Comment:
need a rebase and re-bump of the version

Reply:
This is getting ridiculous.